### ISSUE TITLE
Some of the functionalities are missing. Need implementation.

### ISSUE THREAD

**Issue by raghuprathap** [mnaresh2012/movieApp#1]
Below are the functionalities missing.
1. Result should also provide an add button, to add movies to his lists. By default, only favourites list should be displayed. Data about User's Lists and movies in lists should persist between browser refresh
2. User should be able to add, edit and rename lists
3. Provide a link in the header, to display users list. When a list is selected, display all movies that are part of the list.

**Comment by mnaresh2012** [comment]
Yeah. I am implementing all the missing functionalities. Will update soon. Thanks.

**Comment by raghuprathap** [comment]
if you have worked on these issues please close them.

**Comment by mnaresh2012** [comment]
Sure, @raghuprathap. I will update the repository and close the issues. Thanks.

**Comment by Fawke** [comment]
Hi @mnaresh2012 , did get time to work on the issue, it's been pending since a long time. You can close the issue if you have refactored, and we'll initiate a second review.
